### PR TITLE
generated changelog for 0.14

### PR DESCRIPTION
**Pull request by jasonwilliams**
The changelog was generated from the releases page. Because we don't always use the labels this only picked up some of the changes. Still its not a bad format and I hope to lean on it more going forward.

There's a lot more to fill in but I think we may need to go with this format going forward.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,134 @@
 # CHANGELOG
 
+## What's Changed
+
+# [0.14.0 (TBD) - ](https://github.com/boa-dev/boa/compare/v0.13...v0.14)
```
Comment (jedel1043) [inline CHANGELOG.md:5]: Change TBD to tomorrow's date?

**Review by jedel1043 — APPROVED** [review]
Looks good to me!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,134 @@
...
+- Removing the panic in favour of an error result by @Razican in [#1874](https://github.com/boa-dev/boa/pull/1874)
+- Make `Object.getOwnPropertyDescriptors` spec compliant by @HalidOdat in [#1876](https://github.com/boa-dev/boa/pull/1876)
+- Make `Error` and `%NativeError%` spec compliant by @HalidOdat in [#1879](https://github.com/boa-dev/boa/pull/1879)
+- Fix `Number.prototype.toString` when passing `undefined` as radix by @HalidOdat in [#1877](https://github.com/boa-dev/boa/pull/1877)
+- Cleanup vm stack on function return by @raskad in [#1880](https://github.com/boa-dev/boa/pull/1880)
+- `%NativeError%.[[prototype]]` should be `Error` constructor by @HalidOdat in [#1883](https://github.com/boa-dev/boa/pull/1883)
+- Make `StringToNumber` spec compliant by @HalidOdat in [#1881](https://github.com/boa-dev/boa/pull/1881)
+- Fix `PropertyKey` to `JsValue` conversion by @HalidOdat in [#1886](https://github.com/boa-dev/boa/pull/1886)
+- Make iterator spec complaint by @HalidOdat in [#1889](https://github.com/boa-dev/boa/pull/1889)
+- Implement `Number.parseInt` and `Number.parseFloat` by @HalidOdat in [#1894](https://github.com/boa-dev/boa/pull/1894)
```
Comment (HalidOdat) [inline CHANGELOG.md:91]: Should be in the Feature section.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,134 @@
 # CHANGELOG
 
+## What's Changed
+
+# [0.14.0 (TBD) - ](https://github.com/boa-dev/boa/compare/v0.13...v0.14)
+
+<!-- Release notes generated using configuration in .github/release.yml at main -->
+
+### Feature Enhancements
```
Comment (HalidOdat) [inline CHANGELOG.md:9]: Missing #1831

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,134 @@
...
+- Toggleable `JsValue` internals displaying by @HalidOdat in [#1865](https://github.com/boa-dev/boa/pull/1865)
+- Implement generator execution by @raskad in [#1790](https://github.com/boa-dev/boa/pull/1790)
+- Feature arrays with empty elements by @HalidOdat in [#1870](https://github.com/boa-dev/boa/pull/1870)
+- Removed reference counted pointers from `JsValue` variants by @Razican in [#1866](https://github.com/boa-dev/boa/pull/1866)
+- Implement `Object.prototype.toLocaleString()` by @HalidOdat in [#1875](https://github.com/boa-dev/boa/pull/1875)
+- Implement `AggregateError` by @HalidOdat in [#1888](https://github.com/boa-dev/boa/pull/1888)
+- Implement destructing assignments for assignment expressions by @raskad in [#1895](https://github.com/boa-dev/boa/pull/1895)
+- Added boa examples by @elasmojs in [#1161](https://github.com/boa-dev/boa/pull/1161)
+
+### Bug Fixes
```
Comment (HalidOdat) [inline CHANGELOG.md:61]: Missing #1908

File: CHANGELOG.md (hunk 2 above)
Comment (Razican) [reply CHANGELOG.md:91]: I think this was because it had both the "bug" and "enhancement" tags. I removed the former.

**Pull request merged**